tests/orbit/models/test_ktrlite.py

```diff
@@ -0,0 +1,162 @@
+import pytest
+import numpy as np
+import pandas as pd
+
+from orbit.estimators.stan_estimator import StanEstimatorMAP
+from orbit.models.ktrlite import BaseKTRLite, KTRLiteMAP
+from orbit.diagnostics.metrics import smape
+
+
+@pytest.mark.parametrize("estimator_type", [StanEstimatorMAP])
+def test_ktrlite_single_seas(make_daily_data, estimator_type):
+    train_df, test_df, coef = make_daily_data
+
+    ktrlite = KTRLiteMAP(
+        response_col='response',
+        date_col='date',
+        seasonality=[365.25],
+        seasonality_fs_order=[5],
+        estimator_type=estimator_type
+    )
+
+    ktrlite.fit(train_df)
+    predict_df = ktrlite.predict(test_df)
+
+    expected_columns = ['date', 'prediction_5', 'prediction', 'prediction_95']
+    expected_shape = (364, len(expected_columns))
+    expected_num_parameters = 6
+
+    assert predict_df.shape == expected_shape
+    assert predict_df.columns.tolist() == expected_columns
+    assert len(ktrlite._posterior_samples) == expected_num_parameters
+    assert smape(test_df['response'].values, predict_df['prediction'].values) <= 0.5
+
+
+@pytest.mark.parametrize("estimator_type", [StanEstimatorMAP])
+def test_ktrlite_dual_seas(make_daily_data, estimator_type):
+    train_df, test_df, coef = make_daily_data
+
+    ktrlite = KTRLiteMAP(
+        response_col='response',
+        date_col='date',
+        seasonality=[7, 365.25],
+        seasonality_fs_order=[2, 5],
```

Comment:
this could be parameterize as a set of different seasonality config together with. above